test/stablex/stablex_large_example.js

```diff
@@ -9,7 +9,7 @@ contract("BatchExchange", async accounts => {
   describe("Large Examples [ @skip-on-coverage ]", () => {
     it("ensures hard gas limit on largest possible ring trade ", async () => {
       const batchExchange = await setupGenericStableX(30)
-      const fivePointFiveMillion = 5500000
+      const limit = 6e8
```

Comment:
I like the change here, but maybe we could have been a bit more explicit and said `gasCap` or `gasLimit`.